OracleTuxedo/salt/samples/bankapp/README.md

```diff
@@ -0,0 +1,98 @@
+# Oracle Tuxedo SALT bankapp sample application container image
+
+This sample "bankapp" application extends the Oracle Tuxedo image.
+
+## Prerequisites
+
+This image uses the Oracle Tuxedo core container image `oracle/tuxedo:latest` as its base. Please follow the [Oracle Tuxedo image](https://github.com/oracle/docker-images/tree/main/OracleTuxedo/core) documentation to build the container image before performing the operations described in this document.
+
+## How to build the image
+
+To build the "bankapp" sample application, run:
+
+```shell
+docker build -t tuxedo-bankapp .
+```
+
+## Create and run a container from the image
+
+Use the image built in the preceding section to create a container that runs the application with the following command:
+
+```shell
+docker run -d -p 5955:5955 tuxedo-bankapp
+```
+
+You can review the container logs by using the `docker logs <container_id>` command. The `container_id` can be found by running the `docker ps` command.
+
+## How to test the application
+
+You can test the application using the `curl` HTTP client using the following sample commands. These commands must be run on the same host upon which the container created above is running.
+
+```shell
+TUX_HOSTNAME="127.0.0.1"
+
+curl -X POST -H "Content-type:application/json" http://${TUX_HOSTNAME}:5955/INQUIRY -d '{"ACCOUNT_ID":10000}'
+```
+
+The expected output is
+
+```json
+        "ACCOUNT_ID":   10000,
+        "FORMNAM":      "CBALANCE",
+        "SBALANCE":     "$1456.00"
+```
```

Comment:
````suggestion
```json
{ 
  "ACCOUNT_ID": 10000,
  "FORMNAM": "CBALANCE",
  "SBALANCE": "$1456.00"
}
```
````

Reply:
I'm hoping the `curl` command outputs valid JSON, which has opening and closing braces.

Reply:
Yes, it does. I had omitted the opening and closing braces ({ and }) for brevity. Adding it back to all.